bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/robot/ProblemRobot.scala

```diff
@@ -612,12 +612,16 @@ class ProblemRobot(s: ShellAdvisor) extends Robot {
       }
 
     private val noPIPhoneNumber = when (p.investigators.pi.phone.isEmpty) {
-      new Problem(Severity.Warning, s"No phone number given for ${investigatorFullName(p.investigators.pi, "PI")}. This is for improved user support.",
-        "Overview", s.inOverview{
-          _.editPi(_.Phone.requestFocus)
-        })
+      new Problem(Severity.Warning,
+        s"No phone number given for ${investigatorFullName(p.investigators.pi, "PI")}. This is for improved user support.",
+        "Overview", s.inOverview(_.editPi(_.Phone.requestFocus)))
     }
 
+    private val invalidPIPhoneNumber = (for {
+      badPhone <- p.investigators.pi.phone if badPhone.count(_.isDigit) < 10
```

Comment:
I'm even a little surprised that there isn't a country that uses 9 or fewer digits.